molecule/openstack/molecule.yml

```diff
@@ -31,4 +31,4 @@ verifier:
   # but is not a requirement. For scenario specific tests, add the appropriate
   # file path to the test or test directory below
   additional_files_or_dirs:
-    - ../shared/*
\ No newline at end of file
+    - ../shared/*
```

Comment:
No tests exist in that dir. What is this scenario testing?

I'm also assuming that openstack is not needed for testing localectl, so this entire scenario can likely go away.